x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_add_footer.tsx

```diff
@@ -53,21 +55,40 @@ export const RuleAddFooter = ({
           <></>
         )}
         <EuiFlexItem grow={false}>
-          <EuiButton
-            fill
-            color="success"
-            data-test-subj="saveRuleButton"
-            type="submit"
-            iconType="check"
-            isDisabled={loadingHealthCheck}
-            isLoading={isSaving}
-            onClick={onSave}
-          >
-            <FormattedMessage
-              id="xpack.triggersActionsUI.sections.ruleAddFooter.saveButtonLabel"
-              defaultMessage="Save"
-            />
-          </EuiButton>
+          <EuiFlexGroup justifyContent="flexEnd" gutterSize="m">
+            <EuiFlexItem grow={false}>
+              <EuiButton
+                fill
+                color="primary"
+                data-test-subj="showRequestButton"
+                isDisabled={loadingHealthCheck}
+                isLoading={isSaving}
+                onClick={onShowRequest}
+              >
+                <FormattedMessage
+                  id="xpack.triggersActionsUI.sections.ruleAddFooter.showRequestButtonLabel"
+                  defaultMessage="Show request"
```

Comment:
maybe `Show API request`? cc @lcawl

Reply:
Updated in this commit [1198485](https://github.com/elastic/kibana/pull/180085/commits/1198485fc2f8806aba05cfa7b1eaeadadd3c9076)